test/functional/runtime/redirectWithIdentity.spec.js

```diff
@@ -0,0 +1,127 @@
+/*
+Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { t, ClientFunction, Selector } from "testcafe";
+import createRuntimeFixture from "./helpers/createRuntimeFixture";
+import createNetworkLogger from "./helpers/createNetworkLogger";
+import addHtmlToBody from "./helpers/addHtmlToBody";
+import { SECONDARY_TEST_PAGE } from "./helpers/constants/url";
+import getReturnedEcid from "./helpers/getReturnedEcid";
+
+const networkLogger = createNetworkLogger();
+
+const container = {
```

Comment:
I wonder if we need to abstract this container and replace it with a creator function?

It seems that huge chunks of it will be the same across tests. What do you think J?

Reply:
It makes it easier to use, especially for other engs.

Something similar to the `createRuntimeFixture` helper.

Reply:
I agree. Something like `createContainerSettings(overrides)`, which returns `{...defaultContainer, ...overrides}`.

Reply:
Yes I agree.